<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="hp-pc" ignored="0" name="Test" tests="3" failures="1" timestamp="2023-06-29T09:28:44 IST" time="71.108" errors="0">
  <testcase name="validateLoginFunctionality" time="15.181" classname="com.TestLayer.LoginPageTest"/>
  <testcase name="validateTitle" time="3.414" classname="com.TestLayer.LoginPageTest"/>
  <testcase name="validateUrl" time="0.014" classname="com.TestLayer.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.TestLayer.LoginPageTest.validateUrl(LoginPageTest.java:35)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateUrl -->
</testsuite> <!-- Test -->
